bespin-web/scripts/start-apache.sh

```diff
@@ -5,7 +5,13 @@ python manage.py migrate
 
 create-lando-user.sh
 create-lando-connection.sh
-load-sample-data.sh
+LOADEDSAMPLEDATA=/var/run/bespin-load.txt
```

Comment:
What problem is this solving? The fixtures shouldn't be duplicated. At worst, I think it would clobber modifications to the sample data or possibly related objects in the database, but I'm curious.

If it's necessary to only do this once, storing this state in the /var directory of the web container is not a great solution (the web container is otherwise stateless/ephemeral)